### ISSUE TITLE
test code in cell 8 is commented out

### ISSUE THREAD

**Issue by yashsharan** [llSourcell/wiki_generator_live#3]
In cell 10 we are using the command reset_test_state.run().
However the code for this is in cell 8 which has been commented out
###########
    #Test
    ###########
    #test_data = tf.placeholder(tf.float32, shape=[1, char_size])
    #test_output = tf.Variable(tf.zeros([1, hidden_nodes]))
    #test_state = tf.Variable(tf.zeros([1, hidden_nodes]))
    
    #Reset at the beginning of each test
    #reset_test_state = tf.group(test_output.assign(tf.zeros([1, hidden_nodes])), 
                                #test_state.assign(tf.zeros([1, hidden_nodes])))

    #LSTM
    #test_output, test_state = lstm(test_data, test_output, test_state)
    #test_prediction = tf.nn.softmax(tf.matmul(test_output, w) + b)

I think is should be uncommented
